src/metabase/api/search.clj

```diff
@@ -447,26 +463,28 @@
   to `table_db_id`.
   To specify a list of models, pass in an array to `models`.
   "
-  [q archived created_by table_db_id models]
-  {q            [:maybe ms/NonBlankString]
-   archived     [:maybe ms/BooleanString]
-   table_db_id  [:maybe ms/PositiveInt]
-   models       [:maybe [:or SearchableModel [:sequential SearchableModel]]]
-   created_by   [:maybe ms/PositiveInt]}
+  [q archived created_by table_db_id models verified]
+  {q           [:maybe ms/NonBlankString]
+   archived    [:maybe :boolean]
```

Comment:
by changing this to `:boolean`, this will auto coerce the value to boolean type. this is done by malli under the hood.